### ISSUE TITLE
Tensor Operator Overloading

### ISSUE THREAD

**Issue by robparrish** [robparrish/libgaussian#7]
In the Boost::python export, the operator overloading gets lost. We should either expose this to python (maybe too complicated), or re-implement it in the python side of libpygaussian.

**Comment by jturney** [comment]
This should live in tensor and not libgaussian. I've already started in it there.

**Comment by robparrish** [comment]
Agreed, I just needed something quick. When it is fleshed out in Tensor and
working across both packages, I'll remove it in LibGaussian.

-Rob
